ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -29,16 +31,17 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
     }
 
     private get namespace() {
-        return this.props.match.params.namespace || '';
+        return this.state.namespace;
     }
 
     private set namespace(namespace: string) {
-        document.location.href = uiUrl('archived-workflows/' + namespace);
+        this.setState({namespace});
+        history.pushState(null, '', uiUrl('cron-workflows/' + namespace));
```

Comment:
This edits the URL without forcing a reload. This is cosmetic only, as the source of truth for the namespace is now on react state